.pre-commit-config.yaml

```diff
@@ -1,18 +1,18 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: stable
+    rev: 22.6.0
```

Comment:
why we need to change the versions? 🤔

Reply:
Sorry, I forgot to change it back. For some reason, if I make changes to python scripts Github doesn't let me commit until I have updated the pre-commit. Usually, afterwards I revert the pre-commit back.